tracer/contrib/gocql/gocqltrace.go

```diff
@@ -0,0 +1,147 @@
+// Package gocqltrace provides tracing for the Cassandra Gocql client (https://github.com/gocql/gocql)
+package gocqltrace
+
+import (
+	"context"
+	"github.com/DataDog/dd-trace-go/tracer"
+	"github.com/DataDog/dd-trace-go/tracer/ext"
+	"github.com/gocql/gocql"
+	"strconv"
+	"strings"
+)
+
+// TracedQuery inherits from gocql.Query, it keeps the tracer and the context.
+type TracedQuery struct {
+	*gocql.Query
+	p            traceParams
+	traceContext context.Context
+}
+
+// TracedIter inherits from gocql.Iter and contains a span.
+type TracedIter struct {
+	*gocql.Iter
+	span *tracer.Span
+}
+
+// traceParams containes fields and metadata useful for command tracing
+type traceParams struct {
+	tracer      *tracer.Tracer
+	service     string
+	keyspace    string
+	paginated   string
+	consistancy string
+	query       string
+}
+
+// TraceQuery wraps a gocql.Query into a TracedQuery
+func TraceQuery(service string, tracer *tracer.Tracer, q *gocql.Query) *TracedQuery {
+	string_query := strings.SplitN(q.String(), "\"", 3)[1]
```

Comment:
q.String() returns for a query: (exemple) [query statement="SELECT * from trace.person" values=[] consistency=QUORUM]
So in between the two first " there's the query